### DISCUSSION TITLE
deleting entries by number

### DISCUSSION THREAD

**Discussion by jmgant** [jrnl-org/jrnl#1498] [General]
is it possible to delete an entry based on its number. I have 27 entries, and I want to delete number 24.

**Comment by micahellison** [comment]
Hi @jmgant, it's not. The closest I can think of to this would be running `jrnl --delete` and pressing "y" on the entry you wish to delete and "n" on the others, though that is a bit tedious for 27 entries.

To mitigate that, you could narrow it down to a particular day like so: `jrnl --delete -on 2022-05-05` then press "y" on the entry within that date you wish to delete.
